components/adaptiveEN.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import plotly.graph_objs as go
+from pythermalcomfort.models import adaptive_en
+
+def generate_comfort_chart(tdb, tr, t_running_mean, v):
+    
+    x_values = np.linspace(10, 30,5) 
+    
+    y_values_cat_i_up = []
+    y_values_cat_i_low = []
+    y_values_cat_ii_up = []
+    y_values_cat_ii_low = []
+    y_values_cat_iii_up = []
+    y_values_cat_iii_low = []
+
+    for x in x_values:
```

Comment:
this is not correct, you should not calculalate the the y values for all the x. You just need two x and two y values to draw a line, this is simple math. No need to loop through so many x values. It makes the code slower and the line does not look like a line

Reply:
I think this code only needs two points to draw a straight line. It should be that xy satisfies a linear equation. All I need to do is change "x_values ​​= np.linspace(10, 30, 5)" to "x_values ​​= np.array([10, 30])" in the code. Is there anything wrong with this AdaptiveEN graph?

Reply:
![image](https://github.com/user-attachments/assets/0429dc0d-9c57-4303-aa03-f4bc7e1365bf)
here's the change i made,sir